<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEnglish">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>1031</width>
     <height>87</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Write your English text here</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textGerman">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>1031</width>
     <height>87</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Insert your German text for translation</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>140</y>
     <width>121</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="buttonSynonyms">
      <property name="text">
       <string>Show Synonyms</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSettings">
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonTranslation">
      <property name="text">
       <string>Show Translation</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>891</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCommandLinkButton" name="buttonOption1">
        <property name="text">
         <string>Option1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonOption2">
        <property name="text">
         <string>Option2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonOption3">
        <property name="text">
         <string>Option3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonOption4">
        <property name="text">
         <string>Option4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonOption5">
        <property name="text">
         <string>Option5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCommandLinkButton" name="buttonOption6">
        <property name="text">
         <string>Option6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonOption7">
        <property name="text">
         <string>Option7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonOption8">
        <property name="text">
         <string>Option8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonOption9">
        <property name="text">
         <string>Option9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonOption10">
        <property name="text">
         <string>Option10</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
